### DISCUSSION TITLE
`facetStats` for an attribute shouldn't take into account the filters on that attribute

### DISCUSSION THREAD

**Discussion by xamir82** [meilisearch/product#703] [Feedback & Feature Proposal]
The best example for this would be a `price` attribute and a price range slider on the front-end. When the user sets a min/max value on the range slider, and you send a query with the appropriate filter (e.g. `price >= <min> AND price <= <max>`) to Meilisearch, Meilisearch returns `facetStats` for `price` that's equal to the filter itself. This would be problematic for the front-end, because after the price range slider is updated to match the new `facetStats`, then the user can't change the min & max to greater values.

It seems like you have noticed this problem yourselves in the the official [E-commerce demo](https://ecommerce.meilisearch.com/), where you are performing a multi-search, with two searches, the second one being a dummy search without the filters, and a limit of `1`, as a way of working around the very same issue:

![image](https://github.com/meilisearch/meilisearch/assets/119128472/ec597424-a6a1-42cc-95e2-468357f2717b)

If you don't want to change the current behavior, I think this is certainly the kind of thing that's worth documenting.

**Comment by curquiza** [comment]
Hello @xamir82 
Since it's not a bug but an expected behavior, and we don't accept feature requests on this repo, I move this into the right place so that the product team can manage it 😊 

<img width="1255" alt="Capture d’écran 2023-10-30 à 18 50 04" src="https://github.com/meilisearch/meilisearch/assets/20380692/a5e49af1-d64c-4524-b5cb-bcd54376b22b">

**Comment by curquiza** [comment]
> If you don't want to change the current behavior, I think this is certainly the kind of thing that's worth documenting.

Pinging @meilisearch/devrel-team here since we have a user frustration that could be solved with content creation 😊 
We need discussion with PMs as well if we want to change the behavior instead of relying on content

**Reply by CaroFG** [reply]
Noted! Thank you for the ping!
